api/src/main/java/org/pmiops/workbench/reporting/ReportingUploadServiceDmlImpl.java

```diff
@@ -15,28 +15,28 @@
 import javax.inject.Provider;
 import org.pmiops.workbench.api.BigQueryService;
 import org.pmiops.workbench.config.WorkbenchConfig;
-import org.pmiops.workbench.model.ReportingResearcher;
+import org.pmiops.workbench.model.BqDtoUser;
+import org.pmiops.workbench.model.BqDtoWorkspace;
 import org.pmiops.workbench.model.ReportingSnapshot;
-import org.pmiops.workbench.model.ReportingWorkspace;
 import org.pmiops.workbench.reporting.insertion.DmlInsertJobBuilder;
-import org.pmiops.workbench.reporting.insertion.ResearcherParameter;
-import org.pmiops.workbench.reporting.insertion.WorkspaceParameter;
+import org.pmiops.workbench.reporting.insertion.UserParameterColumn;
+import org.pmiops.workbench.reporting.insertion.WorkspaceParameterColumn;
 import org.springframework.context.annotation.Primary;
 import org.springframework.stereotype.Service;
 
 @Service("REPORTING_UPLOAD_SERVICE_DML_IMPL")
 @Primary
 public class ReportingUploadServiceDmlImpl implements ReportingUploadService {
+
   private static final Logger logger = Logger.getLogger("ReportingUploadServiceInsertQueryImpl");
   private static final long MAX_WAIT_TIME = Duration.ofSeconds(60).toMillis();
 
   private final BigQueryService bigQueryService;
   private final Provider<WorkbenchConfig> workbenchConfigProvider;
 
-  private static final DmlInsertJobBuilder<ReportingResearcher> researcherJobBuilder =
-      ResearcherParameter::values;
-  private static final DmlInsertJobBuilder<ReportingWorkspace> workspaceJobBuilder =
-      WorkspaceParameter::values;
+  private static final DmlInsertJobBuilder<BqDtoUser> userJobBuilder = UserParameterColumn::values;
```

Comment:
We need an instance of the `DmlInsertJobBuilder` for each table. There's no constructor, but this line overrides the single remaining abstract method, providing all the enum instances.

Java is the only language where I'd have to do this. It does not appear possible to express that a class should be both an enum type and extend a generic interface.